README.md

```diff
@@ -90,6 +90,7 @@ Two Chinese version of this list are available [here](https://github.com/alim0x/
 - [Audio Recorder](https://launchpad.net/~audio-recorder/+archive/ubuntu/ppa) - Simple audio recorder available on ubuntu ppa. ![Freeware][Freeware Icon]
 - [Clementine](https://www.clementine-player.org/) - Play numerous lossy and lossless audio formats. [![Open-Source Software][OSS Icon]](https://github.com/clementine-player/Clementine) ![Freeware][Freeware Icon]
 - [Cmus](https://cmus.github.io/) - cmus is a small, fast and powerful console music player for Unix-like operating systems.
+- [DeaDBeeF](http://deadbeef.sourceforge.net/) - DeaDBeeF is a music player for *nix-like systems and OS X. [![Open-Source Software][OSS Icon]](https://github.com/DeaDBeeF-Player/deadbeef) ![Freeware][Freeware Icon]
```

Comment:
Can you provide an escape character "\\" before the "\*".  "\*" is a notation for italic letters. Right now it is not causing any problem, nothing is italic, but it should have an escape character, to prevent weird stuffs to happen in the future